### ISSUE TITLE
[Monitoring] When have 1 more windows nodes Cluster Dashboard gets confusing

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#20077]
Version: Master (v2.3) (5/8/19)
Related: rancher/rancher#20076  

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Create a custom RKE cluster with at least 1 windows worker node
- Enable monitoring, view the Cluster Dashboard

**Result:**
![image](https://user-images.githubusercontent.com/45179589/57408914-c75ad880-719b-11e9-82c3-df0473604a43.png)

Monitoring gets "confusing" for lack of a better word. As expected, monitoring only shows details for linux nodes, but as you can see we have 1.4 of 10 reserved CPU and 0.5 of 2 Used. This is because we cannot monitor windows nodes. If this is the case we need to handle this in some way in the UI. I don't have a good solution in mind, but anything -- a warning of some kind would be nice here to explain why. Additionally, I think that on the Cluster / Project monitoring config page it would be nice to have a warning there as well if and only if there is 1 or more windows nodes detected.

**Comment by lvuch** [comment]
@loganhz could you review this issue, please?

**Comment by davidnuzik** [comment]
Reference https://github.com/rancher/rancher/issues/20076#issuecomment-526366196 - the issue still happens. But maybe this is expected. Testing this is on hold as we await more information in that issue.

**Comment by davidnuzik** [comment]
@loganhz things look good with my testing thus far. I had a question though (not sure if this needs to be a new issue)

When I view the cluster page, there is a dispairty between reserved and used CPU. Is this expected? My understanding is when you set up the rancher agent on a windows node some CPU has to be reserved and so k8s only reports so much CPU actually as usable -- is this why I see this?

![image](https://user-images.githubusercontent.com/45179589/63977082-acbcb000-ca67-11e9-8b1c-913bc1281bc4.png)

**Comment by davidnuzik** [comment]
Based on discussion with Logan this is expected. Some CPU and Memory is reserved for Windows per best practices guides. This is why we see this difference.

As such I'm going to open a _separate_ issue for some sort of UI hint/helper to explain this when a cluster has 1 or more windows nodes. This way people are less likely to get confused when they see this disparity.

**Comment by davidnuzik** [comment]
Version: master-head (v2.3) (8/29/19)

Monitoring now works correctly in the UI and there are no longer any "loading..." green text when viewing the cluster page. I also checked (in other similar issues) today Workloads/Pods/Container metrics. These all work correctly based on my testing.

The dashboard can look a little confusing because we have to reserve some CPU/Memory for the windows host but this is tracked in issue #22531 

Otherwise this is working in the UI and functionally working correctly.

We need to test one last area: check project-level monitoring (currently waiting on a test windows image to accomplish this)

**Comment by davidnuzik** [comment]
Closing based on my last comment. Tests were successful. We will test project-level monitoring separately once an image for Windows is available.
